Assets/MixedRealityToolkit/Inspectors/Profiles/MixedRealityControllerVisualizationProfileInspector.cs

```diff
@@ -185,7 +174,8 @@ private void RenderControllerList(SerializedProperty controllerList)
                 EditorGUILayout.Space();
                 EditorGUILayout.BeginHorizontal();
 
-                EditorGUIUtility.labelWidth = 64f;
+                // TODO: Troy
+                //EditorGUIUtility.labelWidth = 64f;
```

Comment:
Delete these lines

Reply:
This is an outdated file. It has already been removed